src/getter.js

```diff
@@ -102,6 +102,35 @@ function getStoreDeps(getter) {
   return storeDeps
 }
 
+/**
+ * Add override options to a getter
+ * @param {getter} getter
+ * @param {object} options
+ * @param {boolean} options.useCache
+ * @param {*} options.cacheKey
+ * @returns {getter}
+ */
+function addGetterOptions(getter, options) {
+  if (!isKeyPath(getter) && !isGetter(getter)) {
+    throw new Error('createGetter must be passed a keyPath or Getter')
+  }
+
+  if (getter.hasOwnProperty('__options')) {
```

Comment:
Storing these options on `getter` could lead to surprising behavior if `getter` a key path that should be carefully considered and definitely documented. 

A newcomer to nuclear-js might think this would change the cache key (globally):

```js
addGetterOptions(['items'], {cacheKey: "items_cache"});
reactor.evaluate(['items']);
```